ra_aid/proc/interactive.py

```
@@ -1,90 +1,121 @@
 import shutil
-import tempfile
+import errno
+import subprocess
 from typing import List, Tuple
 
-# Add macOS detection
-IS_MACOS = os.uname().sysname == "Darwin"
+import pyte
+from pyte.screens import HistoryScreen
 
+def render_line(line, columns: int) -> str:
```

Ruff (0.8.2)

Missing type annotation for function argument `line`

(ANN001)